### ISSUE TITLE
v4.1.1 - py.test returns 4 errors

### ISSUE THREAD

**Issue by vlauciani** [usgs/shakemap#1222]
Hi

Running `py.test` command on release `v4.1.1`, I receive 4 errors.
```
=========================== short test summary info ============================
ERROR tests/shakemap/coremods/contour_test.py
ERROR tests/shakemap/coremods/kml_test.py
ERROR tests/shakemap/coremods/products_test.py
ERROR tests/shakemap/mapping/mapmaker_test.py
!!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!
```

These errors are the same reported from Azure Pipeline:
- https://dev.azure.com/GHSC-ESI/USGS-ShakeMap/_build/results?buildId=4400&view=logs&j=9f5b6450-9e07-5f46-a12d-62ecc8706db8&t=97df8c6e-129f-5599-4a51-d53ae44b448a&l=168

**Comment by cbworden** [comment]
I'll look into it. It's difficult when things that worked at one time stop working. It usually means a dependency changed, but I'll try to track it down. In the meantime, you might want to try to reinstall and run the tests every day or so. It might "magically" start working again.

**Comment by cbworden** [comment]
This should be fixed now in v4.1.2. Let me know if it is not.
